Gemfile

```diff
@@ -49,20 +50,22 @@ gem 'devise'
 #gem 'devise-guests', '~> 0.3'
 
 # Avalon-specific
-gem 'avalon-workflow', git: "https://github.com/avalonmediasystem/avalon-workflow.git", tag: 'avalon-r6.2'
 gem 'active_encode', '~> 0.1.1'
+gem 'avalon-workflow', git: "https://github.com/avalonmediasystem/avalon-workflow.git", tag: 'avalon-r6.2'
 gem 'hooks'
 gem 'iconv'
-gem 'mediainfo', git: "https://github.com/avalonmediasystem/mediainfo.git", tag: 'avalon-r6.1'
+gem 'mediainfo', git: "https://github.com/avalonmediasystem/mediainfo.git", branch: 'remote_files'
```

Comment:
I think the remote_files branch should get merged into the avalon_fixes branch and then a release cut from there when we're ready for R6.3.  There is one conflict that will need to be resolved before a PR can be made.

Reply:
Resolved with avalonmediasystem/mediainfo#2